Dockerfile

```diff
@@ -26,28 +26,22 @@ RUN apt-get update -y \
         git \
     && python3 -m pip install --upgrade pip \
     && pip3 install umap-learn phate scanpy[leiden] \
+    && pip3 install git+https://github.com/broadinstitute/CellBender.git \
     && mkdir /conga \
     && cd /conga \
     && git clone -b rhesus https://github.com/phbradley/conga.git \
     && cd conga/tcrdist_cpp \
     && make \
-    && cd ../ \
-    && pip3 install -e . \
```

Comment:
@bbimber, I'm testing now on exacloud but I think this is actually necessary for conga to install.